packages/client/test/unit/ServerPersistence.test.ts

```diff
@@ -49,4 +51,22 @@ describe('ServerPersistence', () => {
         await persistence.set('mock-key', 'mock-value', NAMESPACE)
         expect(await persistence.exists()).toBeTrue()
     })
+
+    // enable when NET-1057 done
+    it.skip('concurrency', async () => {
+        const instanceCount = 10
+        const clientId = randomEthereumAddress()
+        const values = await Promise.all(range(instanceCount).map(async (i: number) => {
+            const instance = await ServerPersistence.createInstance({
+                loggerFactory: mockLoggerFactory(),
+                clientId,
+                namespaces: [NAMESPACE],
+                migrationsPath: join(__dirname, '../../src/encryption/migrations')
```

Comment:
Migration files and `NAMESPCE` mismatch

Reply:
Not relevant in this test, but better to unify